Authorization Submission - MoviesMVC

stevietv opened the pull request.

stevietv committed:
import moviesMVC Project

stevietv committed:
scaffold identity authentication

stevietv committed:
implement EnsureCreated

stevietv committed:
switch from ensurecreated to migrate

stevietv committed:
add ErrorLog class and table

stevietv committed:
Add CustomLogger

stevietv committed:
Added Logging to methods

TheCSharpAcademy reviewed (changes requested):
@stevietv Looking good 😎 I just tried a couple of scenarios to test the recording of an error log: I've deleted a record from the database and tried to delete or view it's details but nothing was recorded. Were you able to test this functionality? If so, how can I reproduce? Thanks!

stevietv committed:
Added additional error logging

stevietv commented:
> @stevietv Looking good 😎 I just tried a couple of scenarios to test the recording of an error log: I've deleted a record from the database and tried to delete or view it's details but nothing was recorded. Were you able to test this functionality? If so, how can I reproduce? Thanks!

@TheCSharpAcademy  - Hi Pablo Indeed I missed logging an error when attempting to delete an id that doesn't exist, this is now added.

resulting error table looks like this:

![image](https://github.com/TheCSharpAcademy/CodeReviews.Auth.AuthenticationAndAuthorization/assets/1117625/b264e846-b640-4fbe-8992-489ef061dfe4)
